tests/CMakeLists.txt

```diff
@@ -171,7 +173,7 @@ endif()
 ################################################################################
 # Experimental FuzzTest support (Linux only)
 
-if(AVIF_ENABLE_GTEST AND AVIF_ENABLE_FUZZTEST)
+if(AVIF_ENABLE_FUZZTEST)
```

Comment:
Optional: When the "else" branch is much shorter than the "if" branch, some people recommend inverting the conditional so that the shorter branch goes first. I think that style will make the code much easier to read here, at the cost of testing a negative condition:

```
if(NOT AVIF_ENABLE_FUZZTEST)
  message(STATUS "FuzzTest targets are disabled because AVIF_ENABLE_FUZZTEST is OFF.")
else()
  ... a very long block of code ...
endif()
```